sherlock.h

```diff
@@ -1,3 +1,27 @@
+/*******************************************************************************
+The MIT License (MIT)
+
+Copyright (c) 2015 Dmitry "Dima" Korolev <dmitry.korolev@gmail.com>
```

Comment:
I believe it's missing your name here, Max, even if you feel you haven't touched this particular file yet.